packages/front-end/components/Settings/Team/SingleRoleSelector.tsx

```diff
@@ -48,23 +81,18 @@ export default function SingleRoleSelector({
             role,
           });
         }}
-        options={roleOptions
-          .filter((r) => includeAdminRole || r.id !== "admin")
-          .map((r) => ({
-            label: r.id,
-            value: r.id,
-          }))}
+        options={groupedOptions}
         sort={false}
+        formatGroupLabel={formatGroupLabel}
         formatOptionLabel={(value) => {
           const r = roles.find((r) => r.id === value.label);
+          if (!r) {
+            return;
```

Comment:
Kevin fixed this in a separate PR.  We want to do `return value.label` here instead of returning undefined.